src/pages/home/library/SongList.tsx

```diff
@@ -64,16 +64,35 @@ export default function SongList({ totalCountOfSongs, query }: SongListProps) {
       remainingSongsOnLastPage > 0 ? remainingSongsOnLastPage : rowsPerPage;
   }
 
+  const [streamUrlMissing, setStreamUrlMissing] = useState(true);
```

Comment:
Thank you for the catch! 

For `someSongsNotTranscoded` in the useEffect below it, should I change the variable name as well, since it's a boolean, to `isAnySongsNotTranscoded` or `isAllSongsTranscoded`, it sometimes just sounds strange reading off an if statement with these naming conventions